docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -295,10 +295,12 @@ La configuración anterior genera la siguiente barra lateral:
 
 ### Variantes de insignia
```

Comment:
I think this is missing the heading change with the new "and custom styling" part.